hmpps-kotlin-spring-boot-starter/build.gradle.kts

```diff
@@ -14,7 +14,6 @@ dependencies {
   api("org.springframework.boot:spring-boot-starter-security")
   api("org.springframework.boot:spring-boot-starter-oauth2-client")
   api("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
-  api("org.springframework.boot:spring-boot-starter-cache")
```

Comment:
Not used by the library - warning this is a breaking change, but you'll know about it soon enough.